config/consts.js

```diff
@@ -3,14 +3,19 @@
  */
 
 const APP_INDEX_HTML = 'index.html';
-const APP_INDEX_JS = 'index.js';
+const APP_INDEX_JS = 'app.js';
 
 const MEDIA_FILE_REGEX = /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2)(\?.*)?$/;
 const MEDIA_URL_REGEX = /\.(mp4|webm|wav|mp3|m4a|aac|oga)(\?.*)?$/;
 
+const AUTH0_CLIENT_ID = 'KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH';
+const AUTH0_DOMAIN = 'loom.auth0.com';
+
 module.exports = {
```

Comment:
the top level `config` folder is meant to be for general project configuration, for tasks like building, linting, deploying, etc...

this is application specific, so it belongs under `src`. you can either create a `config` folder under `src`, or something else entirely. up to you.

I'll standardize all of this in `lets-react` at some point, and provide better documentation to provide more clarity.